src/components/pages/Dashboard/Dashboard.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import PageHeader from '../../common/PageHeader/PageHeader';
-import Table from '../../common/Table/Table';
-import Button from '../../common/Button/Button';
+import StyledTable from '../../common/Table/Table';
+import { Header, Button } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+import { getFarmersHandler } from '../../../utils/handlers/farmerHandlers';
 
 const Dashboard = () => {
-  const Title = <h2>Dashboard</h2>;
-  const buttonText = "Add Farmer";
+  const [data, setData] = React.useState([]);
+  const Title = <Header as="h1">All Farmers</Header>;
+  const buttonText = 'Sort By(not functional)';
```

Comment:
💡 I feel like this would just confuse the user => There is a button, but it doesn't do anything. So why is it there in the first place?

I'd suggest removing it.

Reply:
So basically the idea for both buttons was to have already all the styling exactly the same as in the mockup even if the functionality behind was not still ready, but we could remove both. Can you take care of this, @quansenB? I'll take care of the package.json